BigCommerce/Integration/02_Particularities.md

```diff
@@ -0,0 +1,196 @@
+- If you have the *PIM* module not in use, you can export *Omni-Channel* offers directly. 
+- The following is relevant if you started using *BigCommerce* before your started using the *Omni-Channel* module. You can initially import your *BigCommerce* product data to *Omni-Channel*.   
+  Note, if you choose this way, you will not be able to import product changes again from *BigCommerce* to *Actindo*. In this case, you must change the offers on the *Actindo* side and export them to *BigCommerce*.  
+  For this reason, it is recommended to manage offer data at *Actindo*.
+ 
+## Export of offers 
+
+The following particularities are relevant when you export your offers from the *Omni-Channel* module to *BigCommerce*.
+
+### Variant options
+
+*BigCommerce* supports multiple types for the selection of variant options on the UI.  For detailed information, see [Configure BigCommerce connection](./01_ManageBigCommerceConnection.md#configure-bigcommerce-connection).
+
+
+### Custom fields
+
+*BigCommerce* supports simple string values only. For this reason, only the following *Actindo* data types are supported as possible custom field values for attributes:
+
+- Text field
+- String
+- Float 
+- Integer
+- Date time
+
+All data types that are no string values are converted to string. 
+The length of a custom field string is limited to 255 characters. 
+
+**To do**:   
+Check your custom fields if they meet the requirements. 
+If necessary, create new ones. For detailed information on changing and editing of attributes, see [Manage an attribute](../../DataHub/Integration/01_ManageAttributes.md) in the *DataHub* documentation. 
+
+
+### Images
+
+The first image in the corresponding attribute is marked as thumbnail image for the offer. The alt-text of an image will be added as description for that image. Every other field is not supported.
+For detailed information, see [Check product images](./04_ManageProductData.md#check-product-images).  
+ 
+
+
+
+### Videos
+
+Offer videos are not supported by the driver. If you have a strong need to have videos available, contact your MPS consultant at *Actindo*.
+
+
+### Related products
+
+Related products in *BigCommerce* are used differently than in the *Actindo* standard. *Actindo* assigns the related products from *BigCommerce* the *Offer bundle* data type. For detailed information, see [Offer bundle](../../DataHub/UserInterface/04_DataTypeList.md#offer-bundle) in the *DataHub* documentation.   
+In the *Actindo* standard, the offer bundles define offers that complete an offer. For example, a power supply unit that is required in addition to a device and is part of the offer. In *BigCommerce*, on the other hand, the related products are used to present similar offers that may be of additional interest to the customer.  
```

Comment:
In the *Actindo* standard, the **offer bundles** define offers that complete an offer.
Offer bundles oder related products gemeint hier?

Reply:
ist oben drüber erklärt